frontend/components/elements/CourseItem.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+    <li class="course-item card">
+        <div class="card-body">
+            <template v-if="!editing">
+                <BButton
+                    size="sm"
+                    variant="outline-danger"
+                    class="float-right ml-1"
+                    v-confirm="{ action: deleteCourse, text: 'Are you sure you want to delete this course?' }">
```

Comment:
This confirmation comes up 2 times, but is deleted at the first request.

Reply:
there's a ticket for that. it's the same bug for every resource, no need to record it for all of them individually..